src/components/screens/sprint-view-screen.tsx

```diff
@@ -1,3 +1,4 @@
+import { KeyboardReturn } from "@mui/icons-material";
```

Comment:
Any idea why some of the imports are getting reordered in these reviews? I suspect it is related to the linter (biome) and someone not having it set up correctly. Perhaps something worth mentioning in the trainees daily to check everyones is working correctly.